doc/reference/trd-dynamic-process-loading.md

```diff
@@ -32,10 +32,11 @@ from the kernel, they can be updated or added while the Tock kernel continues
 normal operation. Tock provides an interface for supporting application installs
 at runtime.
 
-Dynamically adding applications is comprised of two operations:
+Dynamically adding applications is comprised of three operations:
 
 1. Storing the new process binary.
 2. Loading the new process binary into a Tock process.
+3. Aborting the storing operation.
```

Comment:
To me this is not part of the regular operations for dynamically loading a process.